src/App.js

```diff
@@ -24,25 +27,44 @@ class App extends React.Component {
     const city = e.target.elements.city.value;
     e.preventDefault();
     const api_call = await fetch(
-      `http://api.apixu.com/v1/current.json?key=${API_KEY}&q=${city}` &&
-        `http://api.apixu.com/v1/forecast.json?key=${API_KEY}&q=${city}`
+      `http://api.openweathermap.org/data/2.5/forecast?q=${city}&cnt=8&units=metric&appid=${API_KEY}`
     );
+
     const response = await api_call.json();
     console.log(response);
-    if (city) {
-      this.setState({
-        temperature: response.current.temp_c,
-        hour: response.forecast.forecastday[0].astro,
-        humidity: response.current.humidity,
-        pressure: response.current.pressure_mb,
 
-        icon: response.current.condition.icon,
-        condition: response.current.condition.text,
+    if (response.list) {
+      this.setState({
+        temperature: response.list[0].main.temp,
+        humidity: response.list[0].main.humidity,
+        pressure: response.list[0].main.pressure,
+        icon: response.list[0].weather[0].icon,
+        time: response.list[1].dt_txt,
+        temp: response.list[1].main.temp,
+        icn: response.list[1].weather[0].icon,
+        time: response.list[2].dt_txt,
+        temp: response.list[2].main.temp,
+        icn: response.list[2].weather[0].icon,
+        time: response.list[3].dt_txt,
+        temp: response.list[3].main.temp,
+        icn: response.list[3].weather[0].icon,
+        time: response.list[4].dt_txt,
+        temp: response.list[4].main.temp,
+        icn: response.list[4].weather[0].icon,
+        time: response.list[5].dt_txt,
+        temp: response.list[5].main.temp,
+        icn: response.list[5].weather[0].icon,
+        time: response.list[6].dt_txt,
+        temp: response.list[6].main.temp,
+        icn: response.list[6].weather[0].icon,
+        time: response.list[7].dt_txt,
+        temp: response.list[7].main.temp,
+        icn: response.list[7].weather[0].icon,
```

Comment:
...`weather[0].id` is the id you need to use in your if statement to match up the icons.